proxy/src/proxy.rs

```diff
@@ -263,12 +275,40 @@ fn handle_server_event(
 
     match event {
         ClientFlowEvent::CommandSent {
-            tag,
             handle: _handle,
+            tag,
         } => {
             // TODO: log handle
             trace!(role = "p2s", ?tag, "---> Forwarded command");
         }
+        ClientFlowEvent::CommandRejected {
+            handle: _handle,
+            tag,
+            status,
+        } => {
+            // TODO: log handle
+            trace!(role = "p2s", ?tag, ?status, "---> Aborted command");
+            let status = match status.code() {
+                Some(Code::Alert) => {
+                    // Keep the alert message because it MUST be displayed to the user
+                    Status::Bad {
+                        tag: Some(tag),
+                        code: Code::Alert,
```

Comment:
I think it must be ...

```suggestion
                        code: Some(Code::Alert),
```